Add semantic versioning docs (#1885)
growthbook/growthbook: docs/docs/features/targeting.mdx

- Array of Numbers - useful anytime you have multiple numeric values
- Array of Secure Strings - an array of secure strings useful for passing multiple values that you want to keep secure

> #### String Data Type: Versioning

Author: ```suggestion
#### Semantic Version Targeting
```

> In version 2.2, we introduced support for semantic version targeting. Using the string data type and defining a format, you can leverage semantic versioninng comparisons in your targeting definitions.

Author: ```suggestion
In version 2.2, we introduced support for semantic version targeting. Using the string data type and defining a format, you can leverage semantic versioning comparisons in your targeting definitions.
```

> To create a version formatted string attribute, navigate to **SDK Configurations > Attributes** and either create or edit a String attribute. In the format dropdown, select: **Version string.**

Author: ```suggestion
To leverage this feature, you first need to create a version string attribute. Navigate to **SDK Configurations > Attributes** and create or edit a String attribute. In the format dropdown, select: **Version string.**
```
